### PR TITLE
Ubuntu 22.10 raspberry Pi page update

### PR DESCRIPTION
**Pull request by carkod**
## Done

Update raspberry Pi dowload page for Ubuntu 22.10
## QA

- Go to https://ubuntu-com-12197.demos.haus/download/raspberry-pi and compare with [copy docs](https://docs.google.com/document/d/1KgnSBa8mbR7qwlxxtqbE5a1wXh5GjtVums8TIB7IGVw/edit#)

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12162

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/download/raspberry-pi/index.html [hunk 1]
```diff
@@ -194,6 +199,22 @@ <h3>
         </a>
       </p>
     </div>
+    <div class="col-6">
+      <h3>
+        Ubuntu Server {{ releases.latest.full_version }} LTS
+      </h3>
+      <p>
+        The latest version of Ubuntu with five years of long term support, until {{ releases.latest.eol }}.
```
Comment (minkyngkm) [inline templates/download/raspberry-pi/index.html:207]:
```suggestion
        The latest development release of Ubuntu with nine months of support, until {{ releases.latest.eol }}.
```

**Comment by minkyngkm** [comment]
@carkod Could you please point the feature branch for merging and also rebase the branch?

File: templates/download/raspberry-pi/index.html (hunk 1 above)
Comment (carkod) [reply templates/download/raspberry-pi/index.html]: Thanks, updated.

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Copy looks good, one visual aspect needs tweaking

File: templates/download/raspberry-pi/index.html [hunk 2]
```diff
@@ -46,12 +46,9 @@ <h1>
     <h2 class="u-sv2">
       Download Ubuntu Desktop
     </h2>
-    <p class="p-muted-heading">
-      Recommended desktop
-    </p>
   </div>
   <div class="row">
-    <div class="col-8">
+    <div class="col-6">
```
Comment (sowasred2012) [inline templates/download/raspberry-pi/index.html:51]: The two section are not aligned:

image

Whereas they are in the Download Ubuntu Server section further down the page. This is quite hacky, but we need a way to keep those sections aligned visually while also having "Recommended Desktop" flow correctly on smaller screens.

```suggestion
    <div class="col-6">
        <p class="p-muted-heading">&nbsp;</p>
```

Comment (carkod) [reply templates/download/raspberry-pi/index.html:51]: ok. well, I thought the design was not meant to be done this way, but I don't mind getting some design input if needed. For now, I'll update it this way

**Review by sowasred2012 — APPROVED** [review]
LGTM

**Pull request merged**